### PR TITLE
Formatting and replacing some commands with shell builtins for speed.

### PR DESCRIPTION
**Pull request by jamc-code**
Hey!
    I just formatted some code: quoting variables, replacing some calls of `grep` with bash tests, and some general speed improvements by shortening some of the text filtering. Great program and nothing I did was *necessary* but this module is solid so I figured I'd give it a shot. I benchmarked the changes with `hyperfine` and they shave off about 3ms of runtime.
This commit also cleans up a single space I missed last time. How annoying!

Have a great day,
~ Julia

### CODE REVIEW

File: mullvad_module.sh [hunk 1]
```diff
@@ -50,23 +50,23 @@ VPN_CODES=("${VPN_LOCATIONS[@]}")
 VPN_CODES+=("${COUNTRY_CODES[@]}")
 VPN_LOCATIONS+=("${COUNTRIES[@]}")
 
+MULLVAD_STATUS=$(mullvad status | cut -d' ' -f3)
```
Comment (jamc-code) [inline mullvad_module.sh:53]: this call applies to the entire program, slightly reducing runtime by removing the functions needing to call `mullvad status` individually

File: mullvad_module.sh [hunk 2]
```diff
@@ -50,23 +50,23 @@ VPN_CODES=("${VPN_LOCATIONS[@]}")
...
 
 mullvad_report(){
 # continually reports connection status
 
-	mullvad_status=$(mullvad status)
-	ip_address=$(echo $mullvad_status | grep -oe [0-9].*: | cut -f1 -d:)
+	ip_address=$(mullvad status | cut -d' ' -f7 | cut -d':' -f1)
 
-	if echo $mullvad_status | grep -q 'Connected'; then
+	if [ "$MULLVAD_STATUS" == "Connected"  ]; then
```
Comment (jamc-code) [inline mullvad_module.sh]: bash `test` builtin is a bit faster than `grep`

File: mullvad_module.sh [hunk 3]
```diff
@@ -50,23 +50,23 @@ VPN_CODES=("${VPN_LOCATIONS[@]}")
...
 		if hash geoiplookup 2>/dev/null; then
-			country=$(geoiplookup $ip_address | grep "GeoIP Country" | cut -d' ' -f4 | cut -d, -f1)
-			city=$(geoiplookup $ip_address | grep "GeoIP City" | cut -d, -f5 | sed 's/ //')
+			country=$(geoiplookup "$ip_address" | head -n1 | cut -c24-25)
+			city=$(geoiplookup "$ip_address" | cut -d',' -f5 | sed -n '2{p;q}' | sed 's/ //')
```
Comment (jamc-code) [inline mullvad_module.sh:63]: removing more `grep` for speed, line 63 could also be `city=$(geoiplookup "$ip_address" | head -n2 | tail -n1 | cut -d',' -f5 | sed 's/ //')`, it's about the same speed

**Comment by shervinsahba** [comment]
_"I benchmarked the changes with hyperfine and they shave off about 3ms of runtime."_

That's so rad. I appreciate the changes. Thanks Julia!

FYI, I just overhauled the code to ease a transition to other VPN's.

**Pull request merged**